source/jormungandr/jormungandr/parking_space_availability/car/star.py

```diff
@@ -27,49 +27,36 @@
 # IRC #navitia on freenode
 # https://groups.google.com/d/forum/navitia
 # www.navitia.io
-from __future__ import absolute_import, print_function, unicode_literals, division
 
-import logging
-import pybreaker
-import requests as requests
+from __future__ import absolute_import, print_function, unicode_literals, division
 import jmespath
 
-from jormungandr import cache, app, utils, new_relic
-from jormungandr.parking_space_availability import AbstractParkingPlacesProvider
+from jormungandr.parking_space_availability.car.common_car_park_provider import CommonCarParkProvider
 from jormungandr.parking_space_availability.car.parking_places import ParkingPlaces
 from jormungandr.ptref import FeedPublisher
+from jormungandr import app
 
 DEFAULT_STAR_FEED_PUBLISHER = None
 
 
-class StarProvider(AbstractParkingPlacesProvider):
+class StarProvider(CommonCarParkProvider):
 
     def __init__(self, url, operators, dataset, timeout=1, feed_publisher=DEFAULT_STAR_FEED_PUBLISHER, **kwargs):
 
         self.ws_service_template = url + '/?dataset={}&refine.idparc={}'
```

Comment:
just for the record, as you said, we can probably call the service just once, get the complete list, cache it and use it later

Reply:
Sure. To be discussed